flytekit/clis/flyte_cli/main.py

```diff
@@ -11,14 +12,15 @@
 
 from flytekit import __version__
 from flytekit.clients import friendly as _friendly_client
+from flytekit.clis.auth import credentials as _credentials_access
 from flytekit.clis.helpers import construct_literal_map_from_variable_map as _construct_literal_map_from_variable_map, \
     construct_literal_map_from_parameter_map as _construct_literal_map_from_parameter_map, \
     parse_args_into_dict as _parse_args_into_dict, str2bool as _str2bool
 from flytekit.common import utils as _utils, launch_plan as _launch_plan_common
 from flytekit.common.core import identifier as _identifier
 from flytekit.common.types import helpers as _type_helpers
 from flytekit.common.utils import load_proto_from_file as _load_proto_from_file
-from flytekit.configuration import platform as _platform_config
+from flytekit.configuration import creds as _creds_config, platform as _platform_config
```

Comment:
so I think we should keep the underlying config objects regardless (because they'll be useful in pyflyte), but currently none of these files iirc are used in flyte-cli.  That is, the `flytekit.config` style files are never read by flyte-cli, because it's assumed that users of flyte-cli might not have any flyte repos checked out at all - they may only be an administrator.  Auth has a fair amount to config though, it's pretty awkward to pass in everything as command line args.  What does @matthewphsmith think?

Reply:
I think it should definitely be env var based. You can use the configuration package though to make things easier since the objects are env var aware.

Reply:
I think making it env var based makes it difficult to set-up for an end-user. I'd prefer to have a config that can be easily cloned and written, but I'm happy to move it to a separate directory if it makes sense organizationally.

Reply:
Haytham's argument was that there is a `.aws/config` and a `.kube/config` and a `.gitconfig`.  Why can't there be a .flyte/config?  I think it's cleaner than having to rely on five or six environment variables.  What's the downside that you're worried about Matt?

Reply:
I think we can definitely have `.flyte/config` The only thing is then we need to provide config modification commands (but with click opening up the editor is simple)
Another option is to have `.flyte/config` as an option otherwise explicitly specify all the args